linera-service/tests/end_to_end_tests.rs

```diff
@@ -2,27 +2,66 @@
 // Copyright (c) Zefchain Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+#[cfg(any(feature = "aws", feature = "rocksdb", feature = "scylladb"))]
```

Comment:
If we're not going to test with `memory`, this entire file should probably be tagged `#![cfg(any(feature = "aws", feature = "rocksdb", feature = "scylladb"))]

Reply:
We used Memory so yes that part is eliminated.